db/schema.rb

```diff
@@ -11,24 +11,30 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150407234314) do
+ActiveRecord::Schema.define(version: 20150503235216) do
 
   create_table "bookmarks", force: true do |t|
     t.string   "url"
-    t.integer  "topic_id"
     t.datetime "created_at"
     t.datetime "updated_at"
     t.string   "title"
   end
 
-  add_index "bookmarks", ["topic_id"], name: "index_bookmarks_on_topic_id"
+  create_table "likes", force: true do |t|
+    t.integer  "bookmark_id"
+    t.integer  "user_id"
+    t.datetime "created_at"
+    t.datetime "updated_at"
+  end
 
   create_table "topics", force: true do |t|
     t.string  "title"
     t.text    "description"
     t.integer "user_id"
+    t.integer "bookmark_id"
```

Comment:
No `bookmark_id` column on `topics` table. Instead, have a `topic_id` column on the `bookmarks` table.